### ISSUE TITLE
Settings Sync welcome screen keep showing every time when VSCode open

### ISSUE THREAD

**Issue by kevinchu-work** [shanalikhan/code-settings-sync#1122]
The box "Show this page on startup" was unchecked but it still showing every time. 

🌴 **Visual Studio Code Version :** [ 1.41.1 ] 
🌴 **Code Settings Sync Version :** [ 3.4.3 ] 
🌴 **Standard or Insiders :** [ Mac ]

**Comment by simax** [comment]
I'm seeing the same issue on both Windows (10) and Mac (Catalina).

Code Settings Sync Version : [ 3.4.3 ]
Visual Studio Code Version : [ 1.41.1 ]

**Comment by dsully** [comment]
Same here as well.

**Comment by rebeccathedev** [comment]
Same here. Is there any way to disable the welcome page entirely? Any debug I can provide?

**Comment by rebeccathedev** [comment]
Any movement on this?

**Comment by abryant710** [comment]
I am seeing the same problem. Very frustrating. Is there a fix?

**Comment by kevinchu-work** [comment]
Hi Shanalikhan,

Yeah, it fixed, after spend nearly five days of try and error. I wrote about the fix in comment (below the question <https://stackoverflow.com/questions/62248604/browsersync-on-docker-is-running-correctly-and-able-to-detect-changes-but-brows?noredirect=1#comment110147428_62248604>) and you can clone the whole source code from Github <https://github.com/kevinchu-work/Docker_with_BrowserSync>

If you don’t mind, please give me a star on Github~~~

Enjoy!

Regards,
Kevin

**Comment by abryant710** [comment]
I am also seeing this problem but only on one of my computers. Two are syncing with the same settings but only one always shows the Welcome page when opening VSCode. It also opens when I switch project using the popular 'Project Manager' extension. The checkbox doesn't work which should allow me to never show it again.

**Comment by simax** [comment]
I've now switched to the built in sync settings. https://code.visualstudio.com/docs/editor/settings-sync

**Comment by abryant710** [comment]
> I've now switched to the built in sync settings. https://code.visualstudio.com/docs/editor/settings-sync

Perfect. Thanks @simax. Solved my problem.

**Comment by iamjameswalters** [comment]
I'm still encountering this on VS Code 1.52.1, Settings Sync 3.4.3.

I don't seem to see a built-in VS Code settings sync...? So that solution is a no-go for me. 

Any development here?

**Comment by leahkemp** [comment]
Yep solved my problem too, just more information for others who didn't quite pick up what you were putting down straight away

The fix that worked for me:

- Reset the settings of the "Settings Sync" extension by searching for "Reset Extension Settings" in the command palette
- Upon next startup of VScode, uncheck the "Show this page on startup" button
- On the next startup of VScode the page wasn't shown on startup anymore 😄 

I'm using:

Settings Sync Version : 3.4.3 
Visual Studio Code Version : 1.57.1
Ubuntu 20.04

**Comment by mpiber** [comment]
> Yep solved my problem too, just more information for others who didn't quite pick up what you were putting down straight away
> 
> The fix that worked for me:
> 
> * Reset the settings of the "Settings Sync" extension by searching for "Reset Extension Settings" in the command palette
> * Upon next startup of VScode, uncheck the "Show this page on startup" button
> * On the next startup of VScode the page wasn't shown on startup anymore 😄
> 
> I'm using:
> 
> Settings Sync Version : 3.4.3 Visual Studio Code Version : 1.57.1 Ubuntu 20.04

@leahkemp Thank you very much, it worked for me, i use windows! 💯

**Comment by shriharip** [comment]
Hello, I am still facing this issue. I have the sync settings turned on. Version: 1.63.2. OS Mac OS Montery

**Comment by PickledDev** [comment]
@mpiber those steps worked for me today (30-dec-21).  Thanks!

**Comment by josuerodcat90** [comment]
> Yep solved my problem too, just more information for others who didn't quite pick up what you were putting down straight away
> 
> The fix that worked for me:
> 
> * Reset the settings of the "Settings Sync" extension by searching for "Reset Extension Settings" in the command palette
> * Upon next startup of VScode, uncheck the "Show this page on startup" button
> * On the next startup of VScode the page wasn't shown on startup anymore 😄
> 
> I'm using:
> 
> Settings Sync Version : 3.4.3 Visual Studio Code Version : 1.57.1 Ubuntu 20.04

Thanks a lot!!, worked for me on VSCode latest, W10 and Ubuntu 20.04.

**Comment by hansderly** [comment]
> Yep solved my problem too, just more information for others who didn't quite pick up what you were putting down straight away
> 
> The fix that worked for me:
> 
> * Reset the settings of the "Settings Sync" extension by searching for "Reset Extension Settings" in the command palette
> * Upon next startup of VScode, uncheck the "Show this page on startup" button
> * On the next startup of VScode the page wasn't shown on startup anymore 😄
> 
> I'm using:
> 
> Settings Sync Version : 3.4.3 Visual Studio Code Version : 1.57.1 Ubuntu 20.04

@leahkemp , thanks a lot!
